src/main/kotlin/screeps/api/Constants.kt

```diff
@@ -263,8 +263,19 @@ external val ORDER_BUY: OrderConstant
 external val SUBSCRIPTION_TOKEN: TradableConstant
 
 // Constants to restrict function arguments
```

Comment:
Could you change this comment to something that better reflects that the declarations below are constants that are used as such but never declared in the api.

Reply:
Tried..